frontend/src/views/groups/GroupsView.tsx

```diff
@@ -18,9 +18,11 @@ export const GroupsView = () => {
   const { data: user } = useQuery({
     queryKey: ["groupsData"],
     queryFn: () =>
-      axios.get(ME_URL).then((res: AxiosResponse<User>) => {
-        setUser({ user_id: res.data.user_id });
-        return res.data;
+      axios.get(ME_URL).then(({data: user}: AxiosResponse<User>) => {
+        setUser({ user_id: user.user_id });
+        user.groups.sort((a, b) => a.group_id - b.group_id);
+        user.groups.forEach((group) => group.members.sort((a, b) => a.user_id - b.user_id));
```

Comment:
Instead of sorting by the user's id, it would be better to sort by their name. This is because of a limitation on the backend that will give a lower id to your own user than the rest of the group, regardless of the alphabetical ordering of the names. It happens because your own user must be loaded and stored before any groups can be fetched from OW.